styles/mixins/triangle.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function( config ) {
```

Comment:
I don't follow.  What do you mean its an icon?  Should just be an `:after` on the selected tab state